### ISSUE TITLE
[MAS-Art] Food Items Checklist (In Progress)

### ISSUE THREAD

**Issue by kkrosie123** [Monika-After-Story/MonikaModDev#4471]
**AUGUST 31ST UPDATE:** 
_I'll be working on these! I'm going to add some more options and draw them out! I'll link their issues here when they are done. :)_

So, hello there!

Okay, so I think it would be cool to be able to eat with Monika. I'm sure this has been suggested many, many times. 

I wanted to ask before I started on this because I'm not sure if this would be implemented or not. But anyway, I could work on some food items for Monika. They could easily be replaced by someone else or fixed up by a better artist than myself. But if you would like a placeholder for this feature, I could make the following for Monika:

**All Food Options**
- A club sandwich
- A salad
- A yogurt parfait
- An Apple
- Chili
- Pasta (No meatballs)
- Rice **In progress, no issue created yet**
- Scrambled Eggs (Monika is vegetarian, not vegan :V)

**Possible Breakfast Options**
- A yogurt parfait 
- Scrambled Eggs
_for breakfast._

**Possible Lunch Options**
- A club sandwich
_for lunch or dinner._
- A salad
_for lunch or dinner._
- A yogurt parfait 
_for breakfast or lunch._

**Possible Dinner Options**
- A club sandwich
_for lunch or dinner._
- A salad
_for lunch or dinner._
- Pasta
_for special occasions._
- Chili
_for dinner._

**Possible Snack Options (Low Priority)**
- An Apple
_for a snack._

**Desserts**
- Donuts (Chocolate frosting, of course! I'm thinking rainbow sprinkles!) - In progress! #4580
- Chocolate Cake
- Brownies

If you have any more suggestions, let me know! I'm happy to work on these.

#3174 (Heheh, I don't know if I should tag this or not.)

**Comment by ThePotatoGuy** [comment]
Alright, so first off, there are some food art already made that have yet to be added [search link](https://github.com/Monika-After-Story/MonikaModDev/issues?utf8=✓&q=is%3Aissue+consumable+is%3Aopen+).

Right now, any food is waiting on #3576, which would allow for users to gift food to Monika that should could consume periodically.  There's no eta on this atm, but its likely to be worked on within 2-3 releases, which translates to roughly 2-4 months.

Making food art in the meantime, though, is okay, as long as you understand that's not going to be added right away because of the planned development. If you need help/feedback/etc with making food art, @Professor-JMO would be the best to ask since he has been making food art for a while now.

**Comment by kkrosie123** [comment]
Ah, I see. Thank you for letting me know. UwU

  Alrighty! I see. I will gladly make a few consumables even if it takes a long time for them to be added or if they aren't added at all. Thank you very much for replying. I'll be sure to make some food items. :)

**Comment by kkrosie123** [comment]
_so another one of these has been added_

_**screams**_

_I'm too happy about this aaaaaaaaaaaaa_
